safedelete/managers.py

```diff
@@ -100,6 +101,32 @@ def all(self, **kwargs):
             qs._safedelete_force_visibility = force_visibility
         return qs
 
+    def update_or_create(self, defaults=None, **kwargs):
+        """Regular update_or_create() fails on soft-deleted, existing record with unique constraint on non-id field
```

Comment:
Not a clear description of the behaviour and intentions of the method.

Reply:
Right. Done.